build-image.sh

```diff
@@ -19,9 +20,9 @@ function log() {
 
 log "Pulling old image version"
 if docker pull $IMAGE:$TAG; then
-	CACHE_FROM="--cache-from $IMAGE:$TAG"
+	CACHE_FROM="--cache-from" "$IMAGE:$TAG"
```

Comment:
This will try to execute the `"$IMAGE:$TAG"` command instead of setting `CACHE_FROM` to an array. https://github.com/koalaman/shellcheck/wiki/SC2086#exceptions seems to give some hints.